### DISCUSSION TITLE
Sub-registries and grouping

### DISCUSSION THREAD

**Discussion by itineric** [verdaccio/verdaccio#4918] [Q&A]
Hello,

Is it possible to create some configuration to emulate sub-registries ?
I would like to have 2 registries, a "releases" and a "snapshots", the "maven" way.
Then I would like to have some way configure "grouping":
- snapshots registry gives access to releases registry and some proxies
- releases gives access to some proxies
- proxies is a group that gives access to all external registries

In the documentation, I did not find the way to reference "local packages" as proxies.

**Comment by mbtools** [comment]
Sounds complicated. You can't get this done with package access rules? 

I haven't tried but you should be able to chain several verdaccios as long as they have different domain names:

```
npmjs <-> verdaccio1 <-> verdaccio2
```
